cypress/integration/2_rules.spec.js

```diff
@@ -222,11 +223,24 @@ describe('Rules', () => {
             force: true,
           });
 
-          YAML_RULE_LINES.forEach((line) =>
-            cy
-              .get('[data-test-subj="rule_flyout_yaml_rule"]', TWENTY_SECONDS_TIMEOUT)
-              .contains(line, TWENTY_SECONDS_TIMEOUT)
-          );
+          cy.get('[data-test-subj="rule_flyout_yaml_rule"]')
+            .get('[class="euiCodeBlock__line"]')
+            .each((lineElement, lineIndex) => {
+              if (lineIndex >= YAML_RULE_LINES.length) {
+                return;
+              }
+              let line = lineElement.text().replaceAll('\n', '').trim();
+              let expectedLine = YAML_RULE_LINES[lineIndex];
+
+              // The document ID field is generated when the document is added to the index,
+              // so this test just checks that the line starts with the ID key.
+              if (expectedLine.startsWith('id:')) {
+                expectedLine = 'id:';
+                expect(line, `Sigma rule line  ${lineIndex}`).to.contain(expectedLine);
```

Comment:
Are there two spaces between `line` and `${`? Or does it look like it in the Github UI?

Reply:
it was 2 spaces typo, thanks